lib/kitchen/directions/eob_composite_page_container/v1.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::EobCompositePageContainer
```

Comment:
I am wondering if this implementation is the same as EocCompositePageContainer. If so, do we need a separate EOB class, or could we just use EOC? It would be good to reuse as much code as possible